Automattic/wp-calypso
# Site Settings: Move Export settings to a separate section

This PR moves the Export settings section to a separate chunk, similar to how some of the rest of the settings sections are split. Currently in production we have one `settings` chunk that is over 600K:
![](https://cldup.com/C9kQMlohN5.png)

This PR splits out the Export to a separate section, which reduces the size of the general settings chunk:
![](https://cldup.com/lYJB7BCFf8.png)
![](https://cldup.com/31f3_S8fgl.png)

The size of the general settings chunk will be reduced more as we land #15777.

A potential improvement to this can be preloading the Export chunk when hovering the "Export" link in the Site Tools card. I intend to implement this in a future PR, but I'm planning to land a common preloading tool before that. 

Also, I'm planning to refactor `SiteSettingsComponent`, now that the only section that consumes it is the `General` settings one - but that's slated for a future PR, because I don't want to block this one.

To test:

* Checkout this branch.
* Test the following path with both a JP and WP.com site and verify there are no regressions with them:
  * http://calypso.localhost:3000/settings/export/
  * http://calypso.localhost:3000/settings/export/$site
  * http://calypso.localhost:3000/settings/export/guided/$site
* Verify the Export functionality works properly (for a .com site).
* Verify the General settings are still loaded properly without regressions.
